carapace-server/src/main/java/org/carapaceproxy/server/certificates/DynamicCertificatesManager.java

```diff
@@ -403,7 +375,6 @@ private void createChallengeForCertificateOrder(CertificateData cert, Order orde
                     cert.setState(DNS_CHALLENGE_WAIT);
                     LOG.log(Level.INFO, "Created new TXT DNS challenge-record for domain {0}.", cert.getDomain());
                 } else {
-                    cert.setState(REQUEST_FAILED);
```

Comment:
Pass throw REQUEST_FAILED is just a waste of time... it's already in WAITING stage, it's useless moving from WAITING to REQUEST_FAILED to WATING again...